### DISCUSSION TITLE
New Test Cases

### DISCUSSION THREAD

**Discussion by justincavalli** [TEAMMATES/teammates#12437] [Q&A] [answered]
Hi, I'm a software engineering student learning to contribute to open source projects. Specifically, I'd like to gain some experience in writing a few test cases and submitting them through a PR. Are there any features or areas of the code base which could use more thorough testing?

**Comment by zhaojj2209** [comment, marked as answer]
We would recommend taking a look at #10920. Though #1501 is also open, we do not recommend adding backend test cases at the moment as backend tests will undergo major changes as part of our ongoing database migration.
